phongo_version.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2014-2018 MongoDB, Inc.
```

Comment:
Since this file was just created, the date should start at 2018. This should read "2018-present" per [DRIVERS-329](https://jira.mongodb.org/browse/DRIVERS-329).